src/test/java/com/google/devtools/build/lib/analysis/StarlarkAttrTransitionProviderTest.java

```diff
@@ -514,6 +514,92 @@ public void testOptionConversionCpu() throws Exception {
     assertThat(getConfiguration(Iterables.getOnlyElement(dep)).getCpu()).isEqualTo("armeabi-v7a");
   }
 
+  private void writeReadAndPassthroughOptionsTestFiles() throws Exception {
+    setStarlarkSemanticsOptions("--experimental_starlark_config_transitions=true");
+    writeWhitelistFile();
```

Comment:
I think this line specifically out of date. Need to amend this so the tests still pass and I can import this into Bazel.

Reply:
I've rebased and Buildkite run the tests.

Reply:
Thanks for rebasing, Fredrik. @sdtwigg ready to merge?